bot/constants.py

```diff
@@ -545,12 +543,8 @@ class _URLs(_BaseURLs):
     connect_max_retries = 3
     connect_cooldown = 5
 
-    site_staff: str = "".join([BaseURLs.site_schema, BaseURLs.site, "/staff"])
-    site_paste = "".join(["paste.", BaseURLs.site])
-
-    # Site endpoints
-    site_logs_view: str = "".join([BaseURLs.site_schema, BaseURLs.site, "/staff/bot/logs"])
-    paste_service: str = "".join([BaseURLs.site_schema, "paste.", BaseURLs.site, "/{key}"])
+    paste_service: str = "".join([BaseURLs.site_paste, "/{key}"])
+    site_logs_view: str = "https://pythondiscord.com/staff/bot/logs"
```

Comment:
I wonder if it would make sense to just have this as `site` or `site_public` (or similar) in BaseURLs with value `https://pythondiscord.com`, and then the endpoint can be specified in the place where it's used since that doesn't need to be configurable. With that we could use it for linking to the site for things other than logs in the future if we wanted.

Reply:
I'm fine with the idea, it's just that we need to be picky about the names since they expand real quick to become confusing.
We had it at `site` before, and it got people & me [confused](https://discord.com/channels/267624335836053506/1076619789939781775/1084897789739282462) when it came to usage.

I agree with you on the fact that it makes sense to have a base url, and **only** that needs to be configurable, as I also didn't like having to configure that long url if i ever wanted to use site logs.

However, for now, I'd like to keep it simple like this, as it's litteraly only needed for viewing logs, and if we run into the need of reusable, we'll think of a proper approach.